docker-compose.yml

```
@@ -0,0 +1,20 @@
+services:
+  irods-server:
+    container_name: irods-server
+    image: "wsinpg/ub-18.04-irods-${IRODS_VERSION:-4.2.11}:${DOCKER_TAG:-latest}"
+    restart: always
+    ports:
+      - "1247:1247"
+      - "20000-20199:20000-20199"
+
+  baton:  
+    image: baton
```

The packages in the irods apt repo are built for amd architecture, so platform should be specified for macs with arm processors:
```suggestion
    image: baton
    platform: linux/amd64
```

---

Thank you!